simphony/io/tests/test_data_container_table.py

```diff
@@ -1,15 +1,15 @@
 import unittest
 from contextlib import closing
 
-import tables
-
 from simphony.io.data_container_description import Record
 from simphony.io.data_container_table import DataContainerTable
 from simphony.io.tests.abc_data_container_table_check import (
     ABCDataContainerTableCheck)
 
+try:
+    import tables
 
-class CustomData(tables.IsDescription):
+    class CustomData(tables.IsDescription):
```

Comment:
@itziakos I was not sure how to handle this. These classes need to be disabled somehow since they depend directly from tables,  but ```@unittest.skipIf``` cannot be used here.

I ignore if there is an elegant solution to this problem out there